python/aida_interchange/LDCTimeComponent.py

```diff
@@ -51,4 +51,3 @@ def make_aif_time_component(self, g):
         self.add_literal(g, time_component, AIDA_ANNOTATION.month, self.month, XSD.gMonth)
         self.add_literal(g, time_component, AIDA_ANNOTATION.day, self.day, XSD.gDay)
         return time_component
-        
```

Comment:
I am a little confused.  Did you remove the newline at the end of this file?

Reply:
> I am a little confused. Did you remove the newline at the end of this file?

I believe the previous file didn't have a newline (red diff).  I believe my version removed the last line-- the line which didn't have a newline.  I could try adding the line back in (with a newline) so the only apparent diff is that the old one doesn't have a newline but the new one does?

Reply:
That doesn't really matter.  Removing the line works just as well.  Thanks for the clarification.